redisenterprise/datadog_checks/redisenterprise/check.py

```diff
@@ -160,17 +160,28 @@ def _get_bdb_dict(self, host, port, service_check_tags):
             }
         return bdb_dict
 
-    def _get_events(self, host, port, bdb_dict, service_check_tags, event_limit):
-        """Scrape the LOG endpoint and put all log entries into Datadog events"""
-        evnts = self._api_fetch_json(
-            "logs",
-            service_check_tags,
+    def _get_events(self, host, port, username, password, bdb_dict, service_check_tags, event_limit):
+        """ Scrape the LOG endpoint and put all log entries into Datadog events """
+
+        # We need to use requests to send the get params since the http wrapper does not allow this
+        r = requests.get(
```

Comment:
Thanks for the contribution. Instead of using requests, you can use the `http` requests wrapper from the base check: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/utils/http.py

Reply:
@fanny-jiang the http does not support sending GET parameters, so I need to use requests.  Ran into a similar issue with allow_redirects in line 101 that was approved by the DD team before

Reply:
@hithwen could you please help with the review?

Reply:
Hi @maguec my apologies for misunderstanding. You are correct, the http wrapper does not currently support params. I'll do another review